docs/data-tunnel/03-tunnel-creation.md

```diff
@@ -3,41 +3,57 @@
 Anyone can broadcast `MsgCreateTunnel` to create a new data tunnel on BandChain. The formal specification of the message is given below:
 
 ```protobuf
-message SignalInfo {
+// SignalDeviation is the type for a signal with soft and hard deviation
```

Comment:
in the actual code, we have soft and hard, but this doesn't.

Reply:
Correct but in cli it doesn't